server/apps/immich/src/api-v1/asset/response-dto/map-marker-response.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { AssetEntity } from '@app/infra/entities';
+import { roundToDecimals } from '../../../utils/coordinate.util';
+
+export type MapMarkerResponseDto = [
+  // assetId
+  string,
+  // latitude
+  number,
+  // longitude
+  number,
+];
+
+export function mapAssetMapMarker(asset: AssetEntity, preload: boolean): MapMarkerResponseDto {
+  const lat = asset.exifInfo?.latitude || 0;
+  const lon = asset.exifInfo?.longitude || 0;
+
+  const assetId = preload ? '' : asset.id;
```

Comment:
This sends an empty string for each asset? They doesn't seem ideal.

Reply:
This way the array has always the same length, no matter if `preload` is true or not. This way we need just one DTO and 3 characters `"",` are IMO not worth removing.

Reply:
I don't understand the benefit of the array always being the same length. You have to know if the value is going to be there or not still. 

What do you think about making assetId the third field and making it optional?